_templates/breadcrumbs.html

```diff
@@ -3,5 +3,21 @@
 {% block breadcrumbs_aside %}
 {% if not meta or meta.get('github_url') != 'hide' %}
 {{ super() }}
+
+<style>
+	.godot-edit-guidelines {
+		float: right;
+		clear: both;
+	}
+
+	@media screen and (max-width: 480px) {
+		.godot-edit-guidelines {
+			display: none;
```

Comment:
Why hide it on mobile phones?

Reply:
Because the edit button is hidden at that resolution as well.

Reply:
Would it make sense to treat them differently? Editing anything on GitHub on mobile sounds like an absolute pain, but the guidelines are just another page in the docs, right?

Reply:
I don't think so. This link is an extra for the "Edit on GitHub" button, it's the only reason to add it on every page. And 480px is only a portion of mobile users. Many if not most phones have bigger resolution than that, and so they'll have both the button and the link available.

Reply:
Good point!